File: 0001/P0001.cpp
#include <iostream>

int naturalSum(int n) {
  return n * (n + 1) / 2;
}

int count(int n) {
  --n;
  return 3 * naturalSum(n / 3) + 5 * naturalSum(n / 5) - 15 * naturalSum(n / 15);
}

int main() {
  int t = 0;
  std::cin >> t;
  for (int i = 0; i < t; ++i) {
    int n = 0;
    std::cin >> n;
    int result = count(n);
    std::cout << result << std::endl;
  }

  return 0;
}

File: 0001/P0001.py

def natural_sum(x):
    return x * (x + 1) // 2

def count(n):
    n = n - 1
    return 3 * natural_sum(n // 3) + 5 * natural_sum(n // 5) - 15 * natural_sum(n // 15)

if __name__ == "__main__":
  t = int(input())
  for i in range(t):
    n = int(input())
    result = count(n)
    print(str(result))